tools/regenByteCode.py

```diff
@@ -29,58 +31,93 @@ def run_sub(message, commands, error):
-    ['../out/Jit/test/ch', '-GenerateLibraryByteCodeHeader', '-Intl', '../lib/Runtime/Library/InJavascript/Intl.js'],
-    'Failed to generate INTL 64bit bytecode')
+// Generated Bytecode Header, this file was created by tools/regenByteCode.py
+// This file contains:
+// a) bytecode for Intl library methods implemented in javascript and
+// b) bytecode for other Js library methods, JsBuiltIns, implemented in javascript
 
-bytecode_job('../lib/Runtime/Library/InJavascript/Intl.js.bc.32b.h',
-    ['../out/Jit/test/ch', '-GenerateLibraryByteCodeHeader', '-Intl', '-Force32BitByteCode','../lib/Runtime/Library/InJavascript/Intl.js'],
-    'Failed to generate INTL 32bit bytecode')
+#define JsBuiltIns(VALUE)'''
 
-# JsBuiltin
-print('Generating JsBuiltin Bytecode')
-bytecode_job('../lib/Runtime/Library/JsBuiltin/JsBuiltin.js.nojit.bc.64b.h',
-    ['../out/noJit/test/ch', '-GenerateLibraryByteCodeHeader', '-JsBuiltIn', '-LdChakraLib', '../lib/Runtime/Library/JsBuiltin/JsBuiltin.js'],
-    'Failed to generate noJit 64bit JsBuiltin Bytecode')
+def append_bytecode(header, command, in_path, file_name, error):
+    command_with_file = command[:]
+    command_with_file.append(in_path + file_name)
+    header.write('//Bytecode generated from ' + file_name + '\nconst char Library_Bytecode_')
+    header.write(file_name[:-3])
+    header.flush()
+    job = subprocess.Popen(command_with_file, stdout=header)
+    job.wait()
+    if job.returncode != 0:
+        sys.exit(error)
 
-bytecode_job('../lib/Runtime/Library/JsBuiltin/JsBuiltin.js.nojit.bc.32b.h',
-    ['../out/noJit/test/ch', '-GenerateLibraryByteCodeHeader', '-JsBuiltIn', '-LdChakraLib', '-Force32BitByteCode', '../lib/Runtime/Library/JsBuiltin/JsBuiltin.js'],
+# Regenerate the bytecode
+def bytecode_job(outPath, command, in_path, error):
+    print('Generating bytecode in file ' + outPath)
+    header = open(outPath, 'w')
+    header.write(header_text)
+    files = os.listdir(in_path)
+    files.sort()
+    filtered_files = []
+    for file_name in files:
+        if re.search('.js$', file_name) is not None:
+            if file_name != 'Intl.js':
+                without_extension = file_name[:-3]
+                parts = without_extension.split('_')
+                header.write(' \\\nVALUE(' + parts[0] + ', ' + parts[1] + ', ' +  parts[0] + parts[1].title() + ')')
+                filtered_files.append(file_name)
+
+    header.write('\n\nnamespace js\n{\n\n#ifdef ENABLE_JS_BUILTINS\n\n')
+
+    # generate bytecode for JsBuiltins
+    command_with_chakra_lib = command[:]
+    command_with_chakra_lib.append('-LdChakraLib')
+    command_with_chakra_lib.append('-JsBuiltIn')
+    for file_name in filtered_files:
+            append_bytecode(header, command_with_chakra_lib, in_path, file_name, error)
+
+    # generate bytecode for Intl
+    command.append('-Intl')
+    header.write('#endif\n\n#ifdef ENABLE_INTL_OBJECT\n\n')
+    append_bytecode(header, command, in_path, 'Intl.js', error)
```

Comment:
Intl is done at the end as it's still handled a little differently from the others.